ClientTests/BreachAlertsTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  BreachAlertsTests.swift
+//  ClientTests
+//
+//  Created by Vanna Phong on 5/29/20.
+//  Copyright © 2020 Mozilla. All rights reserved.
+//
```

Comment:
Please use the standard Mozilla license header (copy from another file)

Reply:
Changed across all files I've created. Thanks for pointing that out!

Reply:
Ah thanks garvan, missed that!